internal/cluster/clustersetup/activity_instance_termination_handler.go

```diff
@@ -0,0 +1,114 @@
+// Copyright © 2019 Banzai Cloud
```

Comment:
**Required**:

The
1. file
2. activity
3. types
**MUST** be renamed to include either `deploy` or `install` in the names to conform to the existing naming conventions and explicitly state what the activity does with the instance termination handler subject.